Pipfile

```diff
@@ -18,8 +18,8 @@ pem = "~=21.1.0 "
 python_version = "3.6"
 
 [dev-packages]
-grpcio = "~=1.33.2"
-grpcio-tools = "~=1.33.2"
+grpcio = "~=1.36.1"
+grpcio-tools = "~=1.36.1"
```

Comment:
These lines here are not necessary. The libraries are already listed in the `packages` section what is correct for our library.
Can you delete them as part of this PR please?